guides/common/modules/proc_configuring-dhcpd-to-use-with-server.adoc

```diff
@@ -54,108 +61,144 @@ subnet _192.168.38.0_ netmask _255.255.255.0_ {
 }
 
 omapi-port 7911;
-key omapi_key {
+key _omapi_key_ {
 	algorithm hmac-md5;
-	secret "_My_Secret_";
+	secret "_<key_secret>_";
 };
-omapi-key omapi_key;
+omapi-key _omapi_key_;
 ----
 +
 Note that the `option routers` value is the IP address of your {ProjectServer} or {SmartProxyServer} that you want to use with an external DHCP service.
-. On {ProjectServer}, define each subnet.
+
+. Perform the following steps on {ProjectServer}:
+
+.. Define each subnet.
 Do not set DHCP {SmartProxy} for the defined Subnet yet.
 +
 To prevent conflicts, set up the lease and reservation ranges separately.
 For example, if the lease range is 192.168.38.10 to 192.168.38.100, in the {ProjectWebUI} define the reservation range as 192.168.38.101 to 192.168.38.250.
-. Configure the firewall for external access to the DHCP server:
+
+.. Open the DHCP port in the `firewalld` service:
 +
 [options="nowrap"]
 ----
 # firewall-cmd --add-service dhcp
 ----
-include::snip_make-firewall-settings-persistent.adoc[]
-. On {ProjectServer}, determine the UID and GID of the `foreman` user:
+
+.. Make the changes persistent:
++
+[options="nowrap"]
+----
+# firewall-cmd --runtime-to-permanent
+----
+
+.. Determine both the UID and the primary GID of the `foreman` user:
```

Comment:
This was never correct which makes me think that nobody ever ran this or we relied on world readable files. It must be `foreman-proxy`, not `foreman`.

Reply:
I updated the user/group names.